tests/codemods/test_django_json_response_type.py

```diff
@@ -0,0 +1,92 @@
+from core_codemods.django_json_response_type import DjangoJsonResponseType
+from tests.codemods.base_codemod_test import BaseSemgrepCodemodTest
+from textwrap import dedent
+
+
+class TestDjangoJsonResponseType(BaseSemgrepCodemodTest):
+    codemod = DjangoJsonResponseType
+
+    def test_name(self):
+        assert self.codemod.name() == "django-json-response-type"
+
+    def test_simple(self, tmpdir):
+        input_code = """\
+        from django.http import HttpResponse
+        import json
+
+        def foo(request):
+            json_response = json.dumps({ "user_input": request.GET.get("input") })
+            return HttpResponse(json_response)
+        """
+        expected = """\
+        from django.http import HttpResponse
+        import json
+
+        def foo(request):
+            json_response = json.dumps({ "user_input": request.GET.get("input") })
+            return HttpResponse(json_response, content_type = "application/json")
```

Comment:
Sorry I know this is a relatively small thing but the spaces on either side of the `=` in a kwarg assignment is not considered as stylistically correct and will be flagged by many tools. We should make sure to format it correctly. I think this should be possible via whitespace nodes in the libcst representation.